backend/src/controllers/users.js

```diff
@@ -0,0 +1,12 @@
+const User = require('../models').User;
+
+module.exports = {
+  async findOrCreate(keycloakId, displayName) {
+    return await User.findOrCreate({
+      where: {
+        keycloakId: keycloakId,
+        displayName: displayName
```

Comment:
Should displayName be in the where clause?  
keyCloakId is unique, but display name isn't.  I think you're going to run into issues if someone changes their display Name outside our db.  Move displayName into 'defaults: {displayName: displayName}

Reply:
Good call - updated in upcoming commit